opennlp-tools/src/test/java/opennlp/tools/eval/SourceForgeModelEval.java

```diff
@@ -410,7 +410,7 @@ public void evalParserModel() throws Exception {
       }
     }
 
-    Assert.assertEquals(new BigInteger("312218841713337505306598301082074515847"),
+    Assert.assertEquals(new BigInteger("68039262350771988792233880373220954061"),
```

Comment:
The change of precision in the other tests was small compared to this one. This was the only thing that got my eye from reading the changes here. Pretty much everything else appears to be just s/Math/StrictMath.

Reply:
I will double-check that one and make sure that value is correct.

Reply:
@kinow This number changed a lot because it is an MD5 hash value. Looks like the test is using the parser chunking model to parse the `leipzig/eng_news_2010_300K-sentences.txt` input file. At the end it computes the hash of the parse string and compares it to an expected value.

`StrictMath` must be causing a difference that's visible in the hash value. I'm guessing it's probably like those other tests where it's a small rounding difference somewhere.

I did verify that `68039262350771988792233880373220954061` is the same value on Java 8 and Java 11.

Reply:
Ah, got it @jzonthemtn , had no idea it was a md5 hash. All good then 🥇 👍